woocommerce/google-listings-and-ads
# Remove AttributeMapping new feature inbox notification

This PR removes AttributeMapping's new Feature inbox notification

### Detailed test instructions:


1. Install GLA
2. Go to `wp_options` and set `gla_install_timestamp` as "1669161400"
3. Go to `wc_admin_notes` table and be sure `gla-attribute-mapping-new-feature` is not there, delete it if it's there
4. Go to Woocommerce - Status - Scheduled Actions and run `wc_gla_cron_daily_notes`
5. Go to WooCommerce - Home and verify that "Our latest improvement to the Google Listings & Ads extension: Custom Attribute Mapping" is NOT appearing in the Inbox Notification
6. Verify there is no residual code related to AttributeMapping Inbox Notification


### Additional details:

p1695687924258069-slack-C0187GK80Q1

Also after discussing with @mikkamp we decided to remove this fully, as the feature was created a long time ago.


### Changelog entry

> Fix - Remove AttributeMapping new feature inbox notification
